### ISSUE TITLE
Command history?

### ISSUE THREAD

**Issue by RJGX** [hellzerg/moonshell#2]
Hi, just started using MoonShell and really appreciate what I see so far, but... is there not a command buffer? I can't seem to pull up previously entered commands, and the arrow-keys don't seem to work as they would in the conventional CLI. I know about the **"!!"** entry to execute the last command, but this does not let you edit the command first, nor do I see any way to access/edit/re-submit commands entered prior to that one.
I hope I'm missing something... could you kindly advise? Thank you!

**Comment by hellzerg** [comment]
Yeah, I know, command history is not yet ready. I will try to implement it some time in the future, I currently don't have the time.

Thanks for using MoonShell!

**Comment by RJGX** [comment]
Ok, well thank you for the reply. I think that feature is important, but I understand very well about time constraints. When/if command history is added, MoonShell could definitely be my full-time console program.
